GutenbergObjectPlugin.php

```diff
@@ -28,19 +28,7 @@
 use GutenbergArray\API;
 use GutenbergArray\Hooks;
 
-class GutenbergObjectPlugin {
```

Comment:
I would keep the gutenberg plugin object, it provides structure and gives all the other objects somewhere to live. Pass the other objects in via the constructor and store them as member variables

Reply:
Keep in mind though that this object should take all the other objects in as dependencies, and it shouldn't do anything itself that isn't a method call of its children.